x-pack/test/functional/apps/infra/home_page.ts

```diff
@@ -98,27 +103,30 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
         await pageObjects.infraHome.goToTime(DATE_WITH_DATA);
```

Comment:
There is one more expect in this collapsed part that I skipped, because it seems to call `getWaffleMap` to sync before doing the assertion, while the other tests take other actions after `getWaffleMap` before doing the assertion.